spec/lib/http/request/writer_spec.rb

```diff
@@ -50,11 +50,10 @@
     context "when body is empty" do
       let(:body) { HTTP::Request::Body.new(nil) }
```

Comment:
[Disclaimer: I don't know Ruby.]

Hmm, I wonder if this is distinct from the actually-empty-body case. That is, I would expect that `HTTP::Request::Body.new("")` should still produce a Content-Length: 0. Maybe worth adding a test?

Reply:
Sure, I added a test. `Body.new("")` still adds a Content-Length: 0.

Reply:
Nice.